lib/ramble/ramble/workspace/workspace.py

```diff
@@ -62,7 +62,14 @@
 _active_workspace = None
 
 #: Path where workspaces are stored
-workspace_path = os.path.join(ramble.paths.var_path, 'workspaces')
+workspace_path = ramble.config.get('config:user:workspace_dir')
+if workspace_path:
+    try:
+        workspace_path = os.path.expanduser(workspace_path)
+    except KeyError:
+        workspace_path = os.path.join(ramble.paths.var_path, 'workspaces')
```

Comment:
I think we should remove the default from paths, and push it into the default configs instead.

Reply:
Tell me what you think of the latest change. Should I add a ramble.paths.default_workspace_dirs setting?